CONTRIBUTING.rst

```diff
@@ -0,0 +1,76 @@
+Contributor Guide
+=================
+
+* Thank you for participating!
+* A contribution(pull request) requires to have one `issue <https://github.com/rucio/rucio/issues/new>`_ created.
+* The issue should contain the motivation, modification and expected results (discussions usually happen there).
+* No pull request will be merged without an associated issue (release notes are generated from issues).
+* You should make sure to add your name (and the organisation) to `AUTHORS`_.
+
+A contribution can be either be an **patch**, **feature**, or **hotfix**:
+ * **Patch** include bugfixes and minor changes to the code and goes in Patch releases (usually made on a bi-weekly schedule).
+ * **Feature** include major developments or potentially disruptive changes and goes in Feature release (made multiple times a year).
+ * **Hotfix** are specific patch happening due to the necessity of important fixes and goes in postfix release (made when needed).
+
+Accordingly, the `repository <https://github.com/rucio/rucio/>`_  consists of three different branches:
+ * the **master** branch includes the patch/minor development of the current major version.
+ * the **next** branch includes the development for the next major version.
+ * the **hotfix** branch includes the patch for postfix releases.
```

Comment:
postfix -> hotfix
(I know they are called .post, but the term postfix is not established so I would just call it hotfix due to simplicity)

Reply:
postfix is well established actually  but hotfix is more consistent with the rest of the document

Reply:
Yes, in retrospective we should have called it postfix, it's the better term. But now we have hotfix so I would rather keep it consistent than correct 😄